test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1156,11 +1156,13 @@ class test_ucp_sockaddr_wireup_fail : public test_ucp_sockaddr_wireup {
             UCS_ASYNC_UNBLOCK(&worker->async);
         }
 
-        ucs_time_t deadline = ucs_get_time() + ucs_time_from_sec(timeout);
+        ucs_time_t deadline = ucs::get_deadline(timeout);
```

Comment:
why default timeout is not OK?

Reply:
no need to wait 1 minute, `wait_for_flag` waits 10 seconds

Reply:
done